client/src/layouts/BaseLayout.scss

```diff
@@ -15,8 +17,9 @@ body {
   flex-direction: column;
   max-width: $breakpoint-desktop;
   margin: 0 auto;
-  min-height: 100vh;
+  min-height: 95vh;
```

Comment:
Why did you change this? 🤔

Reply:
At 100vh the page is too high, and adds a sroll bar

Reply:
Fixed in #106.